### DISCUSSION TITLE
How to use locally stored assets/.jpegs for Icons

### DISCUSSION THREAD

**Discussion by ikigai97** [pnp/sp-dev-fx-webparts#2832] [Q&A]
I have been trying to understand how this code loads and then implements the icons that you can select;

![Capture1](https://user-images.githubusercontent.com/107567870/179031351-57fc194e-6051-4448-8791-c9a9b86285a6.PNG)

![image](https://user-images.githubusercontent.com/107567870/179031436-c1c0444e-6b46-4ca4-84f0-fc8ae4c71907.png)

As I would like to use/create some custom Icons to use instead of the icons already provided.

So in summary, how would I change the code so that I can use locally stored .jpegs for the icons?
